### PR TITLE
Added common PR template

### PR DESCRIPTION
**Pull request by xavi160**

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+### Proposed changes
+
+Please explain the changes you made here. Provide a video or screenshot if nessessary.
+
+### Checklist
+
+- [ ] Lint Rules are passed correctly
+- [ ] Code builds correctly
+- [ ] Created or updated tests for the new code
+- [ ] All existing and new tests passed
```
Comment (EyesOnlyNet) [inline PULL_REQUEST_TEMPLATE.md:14]: Personally i think that just the 3rd point is something what we need here. The other ones are already "checked" by our automations. I think we do not need a manual checkbox where we already have an automation for. :thinking:

Comment (xavi160) [reply PULL_REQUEST_TEMPLATE.md:14]: Also, not all projects have eslint or sonar integrated, right?

**Review by CodePapi — APPROVED** [review]

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (xavi160) [reply PULL_REQUEST_TEMPLATE.md:14]: I agree with you @rcidaleassumpo, we can't trust anyone 😄

However, we could start with the `Created or updated tests for the new code` check and see if people are more aware of writing test cases

Comment (EyesOnlyNet) [reply PULL_REQUEST_TEMPLATE.md:14]: I like the compromise to start with just the one entry in the checklist.

**Comment by EyesOnlyNet** [comment]
@xavi160 can we finish (merge / close) this on?

**Pull request closed without merging**